### ISSUE TITLE
Source code is not stored correctly

### ISSUE THREAD

**Issue by mjmeli** [mssubram/updatedShoppingCart#1]
GitHub is intended to be used as centralized source control management and version control. However, this repo seems to store the source code in compressed "zip" files, which does not align with this vision. This approach will make it impossible to track and restore specific changes within the individual source files, drastically decreasing the ability of users to contribute to this high-potential project. Please consider storing source directly into the repository! Thank you, Meghan.

**Comment by mssubram** [comment]
Stop

**Comment by mjmeli** [comment]
![image](https://cloud.githubusercontent.com/assets/6422129/16919505/f4f74080-4cd7-11e6-88ba-656553fce611.png)

See above image. The zip files are considered binary files by the git system and thus diffs and history cannot be tracked. This also means, if we ever need to restore or merge, the entire project must be merged, which will greatly inhibit parallel development.

**Comment by mjmeli** [comment]
@kgeller, what do you think about this issue?

**Comment by kgeller** [comment]
I agree with @mjmeli. He makes a very valid argument for not using a zip file.
